src/main/sensors/initialisation.c

```diff
@@ -88,6 +89,10 @@ bool sensorsAutodetect(void)
     rangefinderInit();
 #endif
 
+#ifdef USE_OPTICALFLOW
+    opticalflowInit();
```

Comment:
Using the function setDeviceConf makes the ordering of Init not important. Once one of them is initialized, the deviceConf will be set, and the other one will not need to set it again.

Reply:
That was related to comments above.
What if user wants to use one device for rangefinding (long-range laser without flow) and another for opticalFlow ?

Reply:
At this moment, this is not supported using the MT lidar because the MT needs to get the range to check whether the optical flow is correct or not (minimum 8 cm), and if you have two separate sensors for range and optical flow and mounted in 2 different levels, then an offset should be configured and this is another story.

However, if the user wants to use the MT as a range finder and another sensor as an optical flow with the current implementation, this will be possible. The other way around is not supported yet but should not fail if you try; the optical flow will not work because the MT RF will not be getting readings. We can handle this by checking the altitude in the optical flow sensor level, not the driver level